extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DevServicesBuildTimeConfig.java

```diff
@@ -54,4 +54,13 @@
      */
     @ConfigItem
     public OptionalInt port;
+
+    /**
+     * Whether this particular data source's container should join TestContainers Shared Network.
+     * <p>
+     * By default, the data source container will join Docker's default 'bridge' network.
```

Comment:
This is a little misleading because the shared network is also a bridge network. The only difference (AFAIK) is that you actually know which network the container is added to.

Reply:
Hi @geoand this is true; and I thought inclusion of "...default..." made it clearer.

However if there is some specific wording you'd prefer I am happy to amend accordingly.

Perhaps if I simply removed "...'bridge'..." it would be clearer?

Reply:
Yeah, that sounds good

Reply:
Done, thank-you @geoand.